.github/workflows/run-tutorial-de.yml

```diff
@@ -0,0 +1,35 @@
+name: run-tutorial-de
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - tutorials/using-edge/**

```

Comment:
@initmahesh , this one should be deploy-edge, Cody asked that we rename it.

Reply:
still needs to be updated

Reply:
Fixed in the PR I am about to create for this feature branch.(I do not have the rights to push a commit directly)